core/pom.xml

```diff
@@ -84,7 +84,7 @@
        -->
       <groupId>org.codehaus.groovy</groupId>
       <artifactId>groovy-all</artifactId>
-      <version>1.8.3</version>
+      <version>2.4.12</version>
```

Comment:
Best shared with the `stapler-groovy` module using `dependencyManagement`.

Reply:
Sure, I'll try and tackle this in some future cleanup PR.

Reply:
That would be nice. I cannot recall offhand if Dependabot is clever enough to file a single PR with both bumps. (I know it does not work in `archetypes` but that is a weird case because the archetype POMs are not technically part of the reactor, and so they have to be added explicitly in Dependabot config sections.)